### DISCUSSION TITLE
Can Macs3 be used to call peaks?

### DISCUSSION THREAD

**Discussion by amderogatis** [GreenleafLab/ArchR#2093] [Questions / Documentation]
Does ArchR support peak calling with the new version of Macs (Macs3) or should Macs2 still be used? Thanks!

**Comment by anaellle** [comment]
Hello ! For me it's working with Macs3, but I had to provide the path manually as the function `findMacs2()` is indeed looking only for Macs2

**Reply by swetansuhota** [reply]
Hi anaellle, 

I am having trouble calling macs3 to archR. Could you please post in detail how you did that...thanks in advance!

Best,
Swetansu

**Reply by anaellle** [reply]
Hi Swetansu, 

If I remember correctly I downloaded macs3, probably added it to my .bash_profile file and just pass the path manually to the peak calling function. But afterwards I managed to install macs2 and ended up using macs2 only. Hope it helps.
